README.md

```diff
@@ -1,14 +1,18 @@
-# go-btrfs
+# go-btrfs (v2)
 
 [![PkgGoDev](https://pkg.go.dev/badge/github.com/containerd/btrfs)](https://pkg.go.dev/github.com/containerd/btrfs)
 [![Build Status](https://github.com/containerd/btrfs/workflows/CI/badge.svg)](https://github.com/containerd/btrfs/actions?query=workflow%3ACI)
 [![Go Report Card](https://goreportcard.com/badge/github.com/containerd/btrfs)](https://goreportcard.com/report/github.com/containerd/btrfs)
 
 Native Go bindings for btrfs.
 
+* v1 (`v1` branch): legacy version with cgo. Used in containerd 1.5 and before.
```

Comment:
For consistency with other projects, I named the branch `release/1.0`

Reply:
Go get probably wants the branch name to be `v1`
https://golang.org/ref/mod#version-queries

> A semantic version prefix, such as v1 or v1.2, which selects the highest available version with that prefix.

Reply:
We already have the `v1.0.0` tag. I don't see from that link how a branch name would be used to resolve the version over semantic version tags

Reply:
Fixed